largest_palindrome_product.js

```diff
@@ -5,18 +5,67 @@
  *  @return {Object} an object containing the two factors used to produce
  *                   the palindromeNumber and the palindromeNumber itself.
  */
-module.exports = function(digits){
+module.exports = function (digits) {
   var factor_0 = 0;
   var factor_1 = 0;
   var palindromeNumber = 0;
 
   // do your work here
+  function getfactor_0() {
```

Comment:
I don't think these three functions are actually necessary. You should be able to access these variables from within the IIFE below.